utils/debug.c

```diff
@@ -38,6 +38,7 @@ const char *color_reset   = TERM_COLOR_RESET;
 const char *color_bold    = TERM_COLOR_BOLD;
 const char *color_string  = TERM_COLOR_MAGENTA;
 const char *color_symbol  = TERM_COLOR_CYAN;
+const char *color_struct  = TERM_COLOR_YELLOW;
```

Comment:
I'm ok with it but I'd like to reserve the yellow for a later use (like a user-defined object name in #632).  What about using CYAN for this too?

Reply:
Hmm.. OK. I can change the color but CYAN is already used by function pointer or global variables so I'm not sure if it's good to add one more category to the same color. How about just using red color instead?

**1. CYAN**
![image](https://user-images.githubusercontent.com/19642328/104836535-665b3180-58a6-11eb-8e5f-e859453773e2.png)

**1. RED**
![image](https://user-images.githubusercontent.com/19642328/104836489-2b58fe00-58a6-11eb-8607-830186b97101.png)

Reply:
Well, personally I prefer cyan (and don't want to make it too colorful).  Maybe we can allow users to customize it later..

Reply:
OK. I've updated it using cyan color.